frontend/src/stores/roles/actions.ts

```diff
@@ -13,9 +13,9 @@ async function fetchRoles(): Promise<void> {
   setRoles(roles)
 }
 
-async function addRole(role: Role): Promise<void> {
-  await post<Role[]>('/roles', role)
+async function addRole(role: Role): Promise<Role[]> {
```

Comment:
We don't need this return anymore, since we are using the exception. Can you revert the modifications in this file?

Reply:
Have you undone the modifications of this file?